frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -39,45 +42,47 @@ class PopupCreateFile extends React.Component {
     props.registerCustomEventHandlerList([
       { name: CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, handler: this.handleAllAppChangeLanguage }
     ])
+
+    props.registerLiveMessageHandlerList([
+      { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.CREATED, optionalSubType: TLM_ST.FILE, handler: this.handleContentCreated }
+    ])
   }
 
-  customEventReducer = ({ detail: { type, data } }) => {
-    switch (type) {
-      case CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE:
-        console.log('%c<PopupCreateFile> Custom event', 'color: #28a745', type, data)
-        this.setState(prev => ({
-          loggedUser: {
-            ...prev.loggedUser,
-            lang: data
-          }
-        }))
-        i18n.changeLanguage(data)
-        this.setHeadTitle()
-        break
-    }
+  handleContentCreated = data => {
+    const { state } = this
+
+    if (data.content.workspace_id !== state.workspaceId || state.loggedUser.user_id !== data.content.author.user_id || !state.fileToUploadList.some(f => f.name === data.content.filename)) return
+
+    const filePosted = new File([data.content], data.content.filename)
+
+    filePosted.jsonResult = { ...data.content, code: 200 }
```

Comment:
hum, why are you hardcoding a response from server that doesn't exist here ?

If you need this code property, I would say this needs a refactor. Because, from here, it looks like a hack